t/udp-generator.pl

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/perl
+
+# Read datagrams from input files and send them to a specified server
+
+use strict;
+use IO::File;
+use IO::Socket;
+
+if (scalar @ARGV < 3) {
```

Comment:
```suggestion
if (@ARGV < 3) {
```

"scalar" can be omitted when comparing against a scalar.